test_cases/inundated_beach_mc/options.py

```diff
@@ -0,0 +1,325 @@
+from thetis import *
```

Comment:
Feel free to overwrite the `inundated_beach` test case rather than creating a new one. I am not using it for anything other than what you are doing.